### DISCUSSION TITLE
Problem Saat Knit to HTML/Word

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#522] [Q&A] [answered]
Dear rekan2 Algoritma,

Saya mendapatkan permasalahan pada saat knit R markdown, sebelumnya berhasil namun saat saya melakukan update package2, saat proses knit gagal.

Capture screen problem:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/103427989/216232067-40e9f3cd-6357-4c33-ae9e-b667acaba1df.png">

versi R yang saya gunakan
<img width="369" alt="image" src="https://user-images.githubusercontent.com/103427989/216232233-4f64e5f8-0876-4fcf-a331-4364c0742283.png">

Best regards,
JT

**Comment by kevbow** [comment, marked as answer]
Dear @joejnt,

Berdasarkan error yang muncul, sepertinya ada masalah pada library `xfun`.
Silakan dicoba untuk remove library ini secara manual lalu lakukan instalasi ulang agar terinstall library dengan versi terbaru.
Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by kevbow** [reply]
Halo @joejnt, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini karena belum ada respon setelah 7 hari respon terakhir. Semoga dapat dipahami.
